### ISSUE TITLE
BN002 should catch invalid folder structure under resources folder

### ISSUE THREAD

**Issue by anaik91** [apigee/apigeelint#552]
In the below proxy folder structure , there are additional folders under `resources/jsc`. This works in OPDK/Edge, but is not allowed in Apigee X/Hybrid.

```
apiproxy/
├── proxy.xml
├── manifests
│ └── manifest.xml
├── policies
│ ├── policy1.xml
│ ├── policy2.xml
| ...
| ...
├── proxies
│ ├── proxy1.xml
├── resources
│ └── jsc
│     ├── script1.js
│     ├── feed
│     │ ├── feed_script1.js
│     │ └── feed_script2.js
│     ├── script2.js
│     ├── menu
│     │ ├── menu_script1.js
│     │ └── menu_script2.js
└── targets
    ├── target1.xml
```

Can we add support for BN002 to catch this for `--profile apigeex`

**Comment by anaik91** [comment]
Closing it , as its supported in Edge as well X/Hybrid. reference : https://cloud.google.com/apigee/docs/api-platform/reference/naming-guidelines
